<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>632</width>
    <height>596</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Hole Measures GUI</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QWidget" name="left_widget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>400</width>
        <height>0</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>4</number>
       </property>
       <property name="leftMargin">
        <number>3</number>
       </property>
       <property name="topMargin">
        <number>3</number>
       </property>
       <property name="rightMargin">
        <number>3</number>
       </property>
       <property name="bottomMargin">
        <number>3</number>
       </property>
       <item>
        <widget class="QPushButton" name="pushButton_test">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Hole Measure Viewer</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_hole">
         <item>
          <widget class="QLabel" name="label_hole">
           <property name="text">
            <string>Hole:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBox_hole">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="toolTip">
            <string>Choose a hole given its index in the TB file</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_TBchoice">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="mouseTracking">
          <bool>false</bool>
         </property>
         <property name="editable">
          <bool>false</bool>
         </property>
         <property name="currentIndex">
          <number>2</number>
         </property>
         <item>
          <property name="text">
           <string>Thickness</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Breadth</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Thickness-Breadth</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_wireframe">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Display wireframe</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_allballs">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>all balls</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="topMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Filter holes:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBox_dim0">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>dim0</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBox_dim1">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>dim1</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBox_dim2">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>dim2</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_minimal_persistence">
           <property name="text">
            <string>Minimal persistence:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="spinbox_min_persistence">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="decimals">
            <number>5</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QSlider" name="slider_min_persistence">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="ChartView" name="chartView_diagram">
         <property name="toolTip">
          <string notr="true">Use arrows to scroll and +/- to zoom in/out</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Hole information:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextBrowser" name="text_HoleInfo"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="MeshViewerWidget" name="displayWidget" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>632</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpenMesh"/>
   <addaction name="actionOpenTB"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpenMesh">
   <property name="text">
    <string>Open mesh</string>
   </property>
  </action>
  <action name="actionOpenTB">
   <property name="text">
    <string>Open TB file</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MeshViewerWidget</class>
   <extends>QWidget</extends>
   <header location="global">meshviewerwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ChartView</class>
   <extends>QGraphicsView</extends>
   <header>chartview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
